manifests/server/config.pp

```diff
@@ -1,10 +1,9 @@
 class puppet::server::config {
 
   include puppet
```

Comment:
If I remove this line, two tests fail, can someone please try it and enlighten me why?